include/dpp/dispatcher.h

```diff
@@ -28,6 +28,11 @@
 
 namespace dpp {
 
+#define API_PATH	"/api/v" DISCORD_API_VERSION
```

Comment:
if we have moved this definition to here, do we then need to take it out of the cpp file it came from? (iirc cluster.cpp)

Reply:
It's just a work around to avoid including the whole cluster.h file, when we only really need some basic stuff from it. But yeah, we can move that away from there.